airbyte-db/lib/src/main/java/io/airbyte/db/instance/configs/ConfigsDatabaseInstance.java

```diff
@@ -4,22 +4,23 @@
 
 package io.airbyte.db.instance.configs;
 
-import com.google.common.annotations.VisibleForTesting;
 import io.airbyte.commons.resources.MoreResources;
 import io.airbyte.db.Database;
 import io.airbyte.db.Databases;
 import io.airbyte.db.ExceptionWrappingDatabase;
 import io.airbyte.db.instance.BaseDatabaseInstance;
 import io.airbyte.db.instance.DatabaseInstance;
 import java.io.IOException;
+import java.util.Collections;
+import java.util.Set;
 import java.util.function.Function;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class ConfigsDatabaseInstance extends BaseDatabaseInstance implements DatabaseInstance {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(ConfigsDatabaseInstance.class);
-
+  private static final Set<String> INITIAL_EXPECTED_TABLES = Collections.singleton("airbyte_configs");
```

Comment:
why do we still need this?

Reply:
@davinchia So the way Flyway is setup is that first we launch a fresh instance of Airbyte,  we first create `airbyte_configs` table and then run all the migrations and eventually reach the latest state. So thats why this is required, we expect this table to be present to determine if the DB has been initialised

Reply:
aye, I get it. annoying. can you leave a comment then? it's not obvious from someone reading code. thanks.